internal/route.go

```diff
@@ -558,7 +558,8 @@ func (routeData *TopicRouteData) decode(data string) error {
 				if i < 0 {
 					continue
 				}
-				id, _ := strconv.ParseInt(str[0:i], 10, 64)
+				brokerId := strings.ReplaceAll(str[0:i], "\"", "")
+				id, _ := strconv.ParseInt(brokerId, 10, 64)
```

Comment:
Can we deserialize JSON string in a standard way? Make the SDK could handle both the two formats.

Reply:
I am afraid we cant use the standard way because the json is not standard, the change have already handled both stardard hand non-stardard formats